rs/nns/governance/src/governance.rs

```diff
@@ -6022,7 +6109,30 @@ impl Governance {
         begin: std::ops::Bound<NeuronId>,
         carry_on: impl FnMut() -> bool,
     ) -> std::ops::Bound<NeuronId> {
-        prune_some_following(&mut self.neuron_store, begin, carry_on)
+        // Hack: Here, we would prefer to use the voting_power_economics method
+        // instead, but if we simply passed self.voting_power_economics() to
```

Comment:
Would this still be a problem if self.voting_power_economics() returned a copy rather than a reference?

Reply:
You are right: we wouldn't need to jump through these extra hoops if it returned a copy. If you are really struggling with references, you can always "escape" by cloning. But you can't go the other way: if you want to avoid copying, but self.voting_power_economics returns one, you are stuck with copying.